tpm2/structures.go

```diff
@@ -104,13 +104,18 @@ func decodePublic(in *bytes.Buffer) (Public, error) {
 // RSAParams represents parameters of an RSA key pair.
 //
 // Symmetric and Sign may be nil, depending on key Attributes in Public.
-// Modulus must always be non-nil.
+//
+// One of Modulus and ModulusRaw must always be non-nil. Modulus takes
+// precedence.
+// ModulusRaw is used for key templates where "unique" field must be a byte
```

Comment:
Please clarify the meaning of the quotes here. Is the name of the field "unique"? If so, then consider rewriting the text as 'where the field named "unique" must be...'. 

Or if this is talking about key templates that have exactly one field, then please drop the quotes and write 'where the unique field must be...'. It might be clearer in this case to write 'where there must be only a single field, consisting of a byte array of all zeroes' or something along those lines.

Reply:
Done, it was a field named `unique` in the spec